### ISSUE TITLE
SSM parameters should be optional

### ISSUE THREAD

**Issue by arturtamborski** [cloudposse/terraform-aws-mq-broker#68]
### Describe the Feature

Hello!

Thank you for this (and other) modules! They are a breeze to use and help us a lot during development.

I would like to propose that SSM parameter creation should be optional. The reason is twofold:
1. always-created SSM parameters is unexpected behavior when compared with rest of your modules (namely: rds, redis)
2. We'd like to manage all SSM parameters in one place (with your ssm module btw), but this module forces us to either duplicate the parameters or split the configuration.

I'm happy to implement this feature if you agree to it and comment with a suggestion for the switch name.

### Expected Behavior

SSM parameter creation should be hidden behind an optional flag. Probably set as a default to false?

### Use Case

We would like to keep all parameters in one place and set them according to our needs, with values sourced from this module's outputs. That's technically still possible, but the module creates its own parameters, causing a bit of confusion and duplication.

### Describe Ideal Solution

SSM parameter creation is hidden behind a flag.
If flag is set to false => no SSM parameter will be created automatically by the module.
If set to true => module works as it does now.

**Comment by sgkouts85** [comment]
Support for this, I am storing passwords in Secrets Manager and don't need them in SSM.

**Comment by elkh510** [comment]
Support for this

**Comment by oycyc** [comment]
Agreed! Put up a PR for this, will get it merged ASAP! This will be backwards compatible as default will be true. https://github.com/cloudposse/terraform-aws-mq-broker/pull/109
